### ISSUE TITLE
Hyperlink meta-preview hover Cards

### ISSUE THREAD

**Issue by kentfredric** [CPAN-API/metacpan-web#1683]
I was just thinking that we're quickly running out of display space in certain interfaces to display metadata about dists.

Specifically, its challenging to add new visual queues to "items" like those proposed in #1682 and #1679 on certain interfaces: 

1. The "Recent" feed 
2. Search results
3. Cross links. ( ??? )

The general idea would be it would only enhance UX for JS clients, by allowing a semi-modal panel to appear in an unobstructive fashion when a hyperlink is hovered over, giving more details about where that hyperlink goes. ( Similar to the current github scorecard popup mechanism )

That way, one could quickly skim quite a few modules in a feed / result set without necessitating each and everyone be opened fully first.

I'll scrub some mock ideas together at some other time, if I don't do it within the week, somebody please ping me.

**Comment by haarg** [comment]
The github hover popup is probably my least favorite part of the UI.  Hovering in general is pretty poor UI, and doesn't work for touch interfaces at all.  Adding simple tooltips to help identify things I think is reasonable, but I wouldn't want to go much further than that.

**Comment by kentfredric** [comment]
Yeah, Doing it right can be hard, and how it works is currently a bit sub-optimal. I think the key is getting it away from the hover point so its not annoying, or having a dedicated icon to click/hover for it.

**Comment by kentfredric** [comment]
As a compromise, how about a ⏬ symbol that injects a short drawer (on row based results) when clicked ( thus not obscuring anything, but shifting things to make room )

**Comment by kentfredric** [comment]
( or an ⓘ )

**Comment by kentfredric** [comment]
![before_click](https://cloud.githubusercontent.com/assets/44790/14938026/7ce12e8e-0f6b-11e6-84d4-2fcf09c740b2.png)
![after_click](https://cloud.githubusercontent.com/assets/44790/14938027/834e9de2-0f6b-11e6-8982-11b5a79bf56a.png)

Some very rough mocks on the "Recent" view. I had an idea that you could possibly show some part of the document in the card, but I'm currently thinking that's a bad idea.
